helpers.cpp

```diff
@@ -27,24 +27,24 @@ void blur(int height, int width, RGBTRIPLE **image) {
     //create a temporary table of colors to not alter the calculations
     RGBTRIPLE tempImg[height][width];
 
-    for (int i = 0; i < width; i++)
+    for (int i {0}; i < width; i++)
     {
```

Comment:
Align brackets properly, please see how brackets were aligned in sepia or edge filter.